espressotee/espresso_tee_helpers_test.go

```diff
@@ -0,0 +1,136 @@
+package espressotee
+
+import (
+	"fmt"
+	"math/big"
+	"testing"
+	"time"
+)
+
+func TestBaseFeeCheck_NilBaseFee(t *testing.T) {
```

Comment:
can we consolidate these tests and make them small, we should not have so many tests just for a nil check I think. Trying to keep our CI times less :)

Reply:
Yes for sure, that's why I added a little comment on slack. I knew this concern would come up 😆

Reply:
[31988a5](https://github.com/EspressoSystems/nitro-espresso-integration/pull/878/commits/31988a5b3e7b07e35d36715b7d2481faea7b7049)